aea/configurations/project.py

```diff
@@ -60,10 +60,12 @@ def load(
         ctx.set_config("skip_consistency_check", skip_consistency_check)
         path = os.path.join(working_dir, public_id.author, public_id.name)
         target_dir = os.path.join(public_id.author, public_id.name)
-        if is_local:
-            fetch_agent_locally(ctx, public_id, target_dir=target_dir)
-        else:
-            fetch_agent(ctx, public_id, target_dir=target_dir)
+
+        if not os.path.exists(target_dir):
```

Comment:
Better to explicitly enable this skip via a default kwarg. Otherwise no error is thrown when we want to.

Reply:
So this is not addressed yet.

Reply:
My bad, now fixed.